src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx

```diff
@@ -1358,6 +1375,18 @@ cordova plugin add https://github.com/newrelic/newrelic-cordova-plugin.git
                 ```
             </td>
         </tr>
+        <tr>
+            <td>
+                Enable or disable offline data storage when no internet connection is available. To configure the amount of offline storage, see [Set max offline storage size](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-offline-storage/#cordova).
+            </td>
+            <td>
+                Offline storage is enabled by default. To disable it, add the following feature flag:
+
+                ```typescript
+                FEDRAMP_ENABLED = "false"
```

Comment:
@ndesai-newrelic Is this supposed to say FEDRAMP?

Reply:
yes